packages/server/src/unstable-core-do-not-import/http/resolveResponse.ts

```diff
@@ -351,6 +351,18 @@ export async function resolveResponse<TRouter extends AnyRouter>(
           type: proc._def.type,
           signal: opts.req.signal,
         });
+
+        if (
+          config.experimental?.outputResponse &&
+          info.isBatchCall &&
+          data instanceof Response
+        ) {
```

Comment:
pretty sure it can't be batched whether you have the flag enabled or not?

```suggestion
        if (info.isBatchCall && data instanceof Response) {
```

Reply:
yeah just wanted to avoid a breaking change